system/nfsexport.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2015 Randy Nance
+# based on code by Antonio Messina, Copyright (C) 2013, GC3, University of Zurich. All rights reserved.
+#
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+
+__docformat__ = 'reStructuredText'
+
+
+DOCUMENTATION = """
+---
+module: nfsexport
+version_added: 2.0
+short_description: This module will configure a nfs exports
+description:
+    - This module will configure an export by writing an entry in the ``/etc/exports`` file.
+    - Please remember to restart the nfs service after updating the ``/etc/exports`` file.
+author: Antonio Messina
+options:
+    path:
+        description:
+            - Directory to export
+        required: true
+    dest:
+        description:
+            - export file to write.
+        required: false
+        default: /etc/exports
+    clients:
+        description:
+            - Address o list of addresses of the nfs clients
+        required: false
+        default: "'*', which means `all`"
+    options:
+        description:
+            - nfs export options.
+        required: false
+        default: ro,root_squash
+        aliases: ['opt', 'opts']
+examples:
```

Comment:
Based on the guidelines[1] this should be in its own section

[1] http://docs.ansible.com/ansible/developing_modules.html#example